modules/deploy/pages/deployment-option/cloud/configure-private-service-connect-in-cloud-ui.adoc

```diff
@@ -0,0 +1,97 @@
+
+* Private Service Connect allows overlapping xref:./cidr-ranges.adoc[CIDR ranges] in VPC networks.
+* Private Service Connect does not limit the number of connections using the service.
+* You control which GCP projects are allowed to connect to the service.
+
+== Requirements
+
+* Use https://cloud.google.com/sdk/docs/install[gcloud^] to create the consumer-side resources, such as a client VPC and forwarding rule, or modify existing resources to use the Private Service Connect service attachment created for your cluster.
+* The client VPC must be in the same region as your Redpanda cluster.
+
+== Enable endpoint service for existing clusters
+
+. In the Redpanda Cloud UI, open your https://cloud.redpanda.com/clusters[cluster^], and click **Cluster settings**.
+. Under Private Service Connect, click **Enable**. 
+. For xref:deploy:deployment-option/cloud/vpc-byo-gcp.adoc[BYOC clusters with customer-managed VPC], you need a NAT subnet with the *Purpose* set to `PRIVATE_SERVICE_CONNECT`. You can create the subnet using the `gcloud` command-line interface (CLI):
++
+[,bash]
+----
+gcloud compute networks subnets create <subnet-name> \
+    --project=<project> \
+    --network=<network-name> \
+    --region=<region> \
+    --range=<subnet-range> \
+    --purpose=PRIVATE_SERVICE_CONNECT
+----
++
+Provide your values for the following placeholders:
++
+- `<subnet-name>`: The name of the NAT subnet.
+- `<project>`: The **host** GCP project ID.
+- `<network-name>`: The name of the VPC being used for your Redpanda Cloud cluster.
+- `<region>`: The region of the Redpanda Cloud cluster.
+- `<subnet-range>`: The CIDR range of the subnet. The mask should be at least `/29`. Each Private Service Connect connection takes up one IP address from the NAT subnet, so the CIDR must be able to accommodate all projects from which connections to the service attachment will be issued.
++
+See the official GCP documentation for https://cloud.google.com/vpc/docs/configure-private-service-connect-producer#add-subnet-psc[creating a subnet for Private Service Connect^].
+. For the accepted consumers list, you need the GCP project IDs from which incoming connections will be accepted.
+. It may take several minutes for your cluster to update. When the update is complete, the Private Service Connect status in **Cluster settings** changes from **In progress** to **Enabled**.
+
+NOTE: For help with issues with enabling Private Service Connect, contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support^].
+
+=== Deploy consumer-side resources
+
+For each VPC network, make sure you also complete the following steps to successfully connect to the service and use Kafka API and other Redpanda services such as HTTP Proxy.
+
+. In **Cluster settings**, copy the **Service attachment URL** under **Private Service Connect**. Use this URL to create the Private Service Connect endpoint in GCP. 
++
+[,bash]
+----
+gcloud 
```

Comment:
@0x5d Are we able to provide the required `gcloud` commands? Or perhaps at least links to the official GCP docs for more guidance? I'm not sure if these are the correct links, or if we need to point readers to specific subsections:
https://cloud.google.com/dns/docs/zones
https://cloud.google.com/dns/docs/records
Also, I couldn't easily find the relevant doc for the service attachment URL step.